src/m365/spo/commands/file/file-add.spec.ts

```diff
@@ -811,16 +788,9 @@ describe(commands.FILE_ADD, () => {
   it('should error if cannot rollback checkout', async () => {
     stubFs();
     const expectedFileAddError: any = JSON.stringify({ "odata.error": { "code": "-2130575338, Microsoft.SharePoint.SPException", "message": { "lang": "en-US", "value": "Error: File add error." } } });
-    const fileAddResp: any = new Promise<any>((resolve, reject) => {
-      return reject(expectedFileAddError);
-    });
-
     const expectedError: any = JSON.stringify({ "odata.error": { "code": "-2130575338, Microsoft.SharePoint.SPException", "message": { "lang": "en-US", "value": "Error: Checkout Error." } } });
-    const rollbackCheckoutResp: any = new Promise<any>((resolve, reject) => {
-      return reject(expectedError);
-    });
 
-    stubPostResponses(null, fileAddResp, null, null, null, rollbackCheckoutResp);
+    stubPostResponses(null, expectedFileAddError, null, null, null, expectedError);
```

Comment:
Oeff, this file brings tears to my eyes. We should definitely refactor this one a bit. But that's not your responsibility today I guess 😀

Reply:
Tell me about it... The old spec files are horrible